### ISSUE TITLE
[Config Support]: TPU not found when custom model configured

### ISSUE THREAD

**Issue by jamesgid-gc** [blakeblackshear/frigate#9444]
### Describe the problem you are having

Hi team, I have Frigate+ and have uploaded approx 50 images and created a custom model. When I alter my config to include this line:
`
model:
  path: plus://*the model id to my edgetpu model
`
  
When I restart my docker container, I see the following:
'
2024-01-26 12:10:42.161231923  [2024-01-26 12:10:42] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found
2024-01-26 12:10:42.161238103  [2024-01-26 12:10:42] frigate.detectors.plugins.edgetpu_tfl ERROR   : No EdgeTPU was detected. If you do not have a Coral device yet, you must configure CPU detectors.
2024-01-26 12:10:42.162096546  ValueError: Could not open 'plus://***my edgetpu model_id***'.
'

So after altering the config using the instructions found here: https://deploy-preview-6262--frigate-docs.netlify.app/integrations/plus/#use-models I get some new errors in my log and the docker container enters a boot loop.

### Version

0.12.1-367D724

### Frigate config file

```yaml
mqtt:
  host: 192.168.0.106
  port: 1883
  user: root
  password: ***password***
  topic_prefix: frigate
  client_id: frigate
  stats_interval: 60

detectors:
  coral:
    type: edgetpu
    device: usb

ffmpeg:
  input_args: preset-rtsp-restream

objects:
  track:
    - person
  filters:
    person:
      min_score: 0.78

motion:
  mask: 0,0,121,0,119,12,83,12,86,24,0,25

record:
  enabled: True
  retain:
    days: 7
    mode: motion
  events:
    pre_capture: 7
    post_capture: 10
    retain:
      default: 14
      mode: active_objects

snapshots:
  enabled: True

go2rtc:
  streams:
    Cam_FrontDoor:
      - rtsp://jamesgid:***password***@192.168.0.120:554/s0
    Cam_FrontGarage:
      - rtsp://jamesgid:***password***@192.168.0.125:554/s0
    Cam_Undercover:
      - rtsp://jamesgid:***password***!@192.168.0.121:554/s0
    Cam_Side:
      - rtsp://jamesgid:***password***@192.168.0.122:554/s0
    Cam_Pool:
      - rtsp://jamesgid:***password***@192.168.0.123:554/s0
    Cam_Backyard:
      - rtsp://jamesgid:***password***@192.168.0.124:554/s0

cameras:
  Cam_FrontDoor: 
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Cam_FrontDoor
          roles:
            - detect
            - record
    detect:
      width: 1280
      height: 720
      fps: 5
    zones:
      driveway:
        coordinates: 0,720,1280,720,1280,276,1115,228,954,187,714,148,525,164,390,167,381,105,0,146
        objects:
          - person
          - car
    record:
      events:
        required_zones:
          - driveway
    snapshots:
      required_zones:
        - driveway
    objects:
      track:
        - person
        - car

  Cam_FrontGarage: 
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Cam_FrontGarage
          roles:
            - detect
            - record
    detect:
      width: 1280
      height: 720
      fps: 5
    zones:
      drivewayLeft:
        coordinates: 0,720,1280,720,1280,180,899,111,517,153,0,290
        objects:
          - person
          - car

          
    record:
      events:
        required_zones:
          - drivewayLeft
    snapshots:
      required_zones:
        - drivewayLeft
    objects:
      track:
        - person
        - car

  Cam_Undercover: 
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Cam_Undercover
          roles:
            - detect
            - record
    detect:
      width: 1280
      height: 720
      fps: 5

  Cam_Side: 
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Cam_Side
          roles:
            - detect
            - record
    detect:
      width: 1280
      height: 720
      fps: 5
    zones:
      sideWalkway:
        coordinates: 0,720,1280,720,1280,610,57,106,0,124
        objects:
          - person
    record:
      events:
        required_zones:
          - sideWalkway
    snapshots:
      required_zones:
        - sideWalkway

  Cam_Pool: 
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Cam_Pool
          roles:
            - detect
            - record
    detect:
      width: 1280
      height: 720
      fps: 5
    zones:
      in_pool:
        coordinates: 1280,181,893,101,821,180,431,142,163,233,104,298,353,514,932,720,1280,622
        objects:
          - person
    record:
      events:
        required_zones:
          - in_pool
    snapshots:
      required_zones:
        - in_pool

  Cam_Backyard: 
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Cam_Backyard
          roles:
            - detect
            - record
    detect:
      width: 1280
      height: 720
      fps: 5
```

### Relevant log output

```shell
2024-01-26 12:10:42.161010927  Process detector:coral:
2024-01-26 12:10:42.161231923  [2024-01-26 12:10:42] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found
2024-01-26 12:10:42.161238103  [2024-01-26 12:10:42] frigate.detectors.plugins.edgetpu_tfl ERROR   : No EdgeTPU was detected. If you do not have a Coral device yet, you must configure CPU detectors.
2024-01-26 12:10:42.161963860  Traceback (most recent call last):
2024-01-26 12:10:42.162002756    File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
2024-01-26 12:10:42.162004302      self.run()
2024-01-26 12:10:42.162006115    File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
2024-01-26 12:10:42.162007958      self._target(*self._args, **self._kwargs)
2024-01-26 12:10:42.162010089    File "/opt/frigate/frigate/object_detection.py", line 98, in run_detector
2024-01-26 12:10:42.162014127      object_detector = LocalObjectDetector(detector_config=detector_config)
2024-01-26 12:10:42.162015864    File "/opt/frigate/frigate/object_detection.py", line 52, in __init__
2024-01-26 12:10:42.162035359      self.detect_api = create_detector(detector_config)
2024-01-26 12:10:42.162037141    File "/opt/frigate/frigate/detectors/__init__.py", line 24, in create_detector
2024-01-26 12:10:42.162038624      return api(detector_config)
2024-01-26 12:10:42.162040215    File "/opt/frigate/frigate/detectors/plugins/edgetpu_tfl.py", line 39, in __init__
2024-01-26 12:10:42.162079629      self.interpreter = Interpreter(
2024-01-26 12:10:42.162093390    File "/usr/lib/python3/dist-packages/tflite_runtime/interpreter.py", line 348, in __init__
2024-01-26 12:10:42.162095152      _interpreter_wrapper.CreateWrapperFromFile(
2024-01-26 12:10:42.162096546  ValueError: Could not open 'plus://***my edgetpu model_id***'.

2024-01-26 12:10:42.754239001  192.168.0.87 - - [26/Jan/2024:12:10:42 +1100] "GET /api/config/raw HTTP/1.1" 200 916 "http://192.168.0.100:5000/config" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36" "-"
2024-01-26 12:10:42.996926807  192.168.0.87 - - [26/Jan/2024:12:10:42 +1100] "GET //api/config/schema.json HTTP/1.1" 200 25729 "http://192.168.0.100:5000/assets/yaml.worker.bundle.js" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36" "-"
2024-01-26 12:10:45.651018769  192.168.0.111 - - [26/Jan/2024:12:10:45 +1100] "GET /api/stats HTTP/1.1" 200 3639 "-" "HomeAssistant/2024.1.5 aiohttp/3.9.1 Python/3.11" "-"
2024-01-26 12:10:48.524495296  [INFO] Starting go2rtc healthcheck service...
```

### Frigate stats

```json
// 20240126121943
// http://192.168.0.100:5000/api/stats

{
  "Cam_Backyard": {
    "camera_fps": 5.1,
    "capture_pid": 476,
    "detection_enabled": 1,
    "detection_fps": 6.5,
    "ffmpeg_pid": 490,
    "pid": 449,
    "process_fps": 5.0,
    "skipped_fps": 0.0
  },
  "Cam_FrontDoor": {
    "camera_fps": 5.1,
    "capture_pid": 450,
    "detection_enabled": 1,
    "detection_fps": 10.4,
    "ffmpeg_pid": 457,
    "pid": 441,
    "process_fps": 5.1,
    "skipped_fps": 0.0
  },
  "Cam_FrontGarage": {
    "camera_fps": 5.0,
    "capture_pid": 454,
    "detection_enabled": 1,
    "detection_fps": 0.1,
    "ffmpeg_pid": 460,
    "pid": 443,
    "process_fps": 5.0,
    "skipped_fps": 0.0
  },
  "Cam_Pool": {
    "camera_fps": 5.0,
    "capture_pid": 472,
    "detection_enabled": 1,
    "detection_fps": 1.8,
    "ffmpeg_pid": 478,
    "pid": 448,
    "process_fps": 5.0,
    "skipped_fps": 0.0
  },
  "Cam_Side": {
    "camera_fps": 5.0,
    "capture_pid": 464,
    "detection_enabled": 1,
    "detection_fps": 6.8,
    "ffmpeg_pid": 470,
    "pid": 446,
    "process_fps": 5.0,
    "skipped_fps": 0.0
  },
  "Cam_Undercover": {
    "camera_fps": 5.0,
    "capture_pid": 458,
    "detection_enabled": 1,
    "detection_fps": 2.0,
    "ffmpeg_pid": 466,
    "pid": 444,
    "process_fps": 5.1,
    "skipped_fps": 0.0
  },
  "cpu_usages": {
    "1": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "15": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "16": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "24": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "25": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "26": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "27": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "28": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "29": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "30": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "31": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "32": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "41": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "42": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "80": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "81": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "82": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "89": {
      "cpu": "9.3",
      "mem": "0.1"
    },
    "97": {
      "cpu": "19.3",
      "mem": "0.6"
    },
    "98": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "112": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "134": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "135": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "136": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "137": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "144": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "164": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "193": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "228": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "426": {
      "cpu": "0.0",
      "mem": "0.2"
    },
    "432": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "433": {
      "cpu": "11.0",
      "mem": "0.3"
    },
    "435": {
      "cpu": "1.3",
      "mem": "0.3"
    },
    "441": {
      "cpu": "0.3",
      "mem": "0.3"
    },
    "442": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "443": {
      "cpu": "0.7",
      "mem": "0.3"
    },
    "444": {
      "cpu": "6.3",
      "mem": "0.3"
    },
    "445": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "446": {
      "cpu": "15.0",
      "mem": "0.3"
    },
    "447": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "448": {
      "cpu": "9.0",
      "mem": "0.3"
    },
    "449": {
      "cpu": "13.6",
      "mem": "0.3"
    },
    "450": {
      "cpu": "1.0",
      "mem": "0.3"
    },
    "451": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "454": {
      "cpu": "1.0",
      "mem": "0.3"
    },
    "457": {
      "cpu": "42.9",
      "mem": "0.2"
    },
    "458": {
      "cpu": "1.0",
      "mem": "0.3"
    },
    "459": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "460": {
      "cpu": "42.5",
      "mem": "0.2"
    },
    "464": {
      "cpu": "1.0",
      "mem": "0.3"
    },
    "466": {
      "cpu": "41.2",
      "mem": "0.2"
    },
    "467": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "470": {
      "cpu": "44.2",
      "mem": "0.2"
    },
    "472": {
      "cpu": "1.0",
      "mem": "0.3"
    },
    "475": {
      "cpu": "0.0",
      "mem": "0.1"
    },
    "476": {
      "cpu": "1.3",
      "mem": "0.3"
    },
    "478": {
      "cpu": "45.2",
      "mem": "0.2"
    },
    "490": {
      "cpu": "47.8",
      "mem": "0.2"
    },
    "1421": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "1447": {
      "cpu": "0.0",
      "mem": "0.0"
    },
    "1451": {
      "cpu": "0.3",
      "mem": "0.0"
    },
    "1453": {
      "cpu": "1.7",
      "mem": "0.1"
    },
    "%Cpu(s):": {
      "cpu": "id,",
      "mem": "0.4"
    },
    "MiB": {
      "cpu": "11254.1",
      "mem": "avail"
    },
    "PID": {
      "cpu": "%CPU",
      "mem": "%MEM"
    },
    "Tasks:": {
      "cpu": "stopped,",
      "mem": "0"
    },
    "top": {
      "cpu": "users,",
      "mem": "load"
    }
  },
  "detection_fps": 27.6,
  "detectors": {
    "coral": {
      "detection_start": 1706231975.514482,
      "inference_speed": 9.17,
      "pid": 433
    }
  },
  "service": {
    "last_updated": 1706231978,
    "latest_version": "0.12.1",
    "storage": {
      "/dev/shm": {
        "free": 123.0,
        "mount_type": "tmpfs",
        "total": 134.2,
        "used": 11.2
      },
      "/media/frigate/clips": {
        "free": 8139348.5,
        "mount_type": "nfs",
        "total": 17269122.7,
        "used": 9129774.2
      },
      "/media/frigate/recordings": {
        "free": 8139348.5,
        "mount_type": "nfs",
        "total": 17269122.7,
        "used": 9129774.2
      },
      "/tmp/cache": {
        "free": 74842.5,
        "mount_type": "overlay",
        "total": 105089.3,
        "used": 24861.2
      }
    },
    "temperatures": {
      
    },
    "uptime": 475,
    "version": "0.12.1-367d724"
  }
}
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment]
Frigate+ models are only available in 0.13 like it says in the docs

**Comment by jamesgid-gc** [comment]
> Frigate+ models are only available in 0.13 like it says in the docs

Oh, I thought I recently updated to the latest. Completely my bad. Thanks 👍
